src/main/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/MessageHelpers.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.helpers
+
+import play.api.i18n.{Lang, Messages, MessagesApi, MessagesImpl}
+
+object MessageHelpers {
+  def messagesForLanguage(
+    code: Option[String] = None
+  )(implicit messages: Messages, messagesApi: MessagesApi): Messages =
+    // TODO: PR question: should I make this a class and @inject messagesApi and pass helper via DI?
```

Comment:
I feel like I prefer making it a class and injecting messages Api rather than asking for it as an implicit, but I hadn't found `Play.current.injector.instanceOf[]` yet to allow for https://github.com/hmrc/play-frontend-hmrc/blob/master/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/html/components/package.scala